modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts

```diff
@@ -347,7 +347,7 @@ export interface IBaseCoin {
   supportsStaking(): boolean;
   supportsBlsDkg(): boolean;
   getBaseFactor(): number | string;
-  baseUnitsToBigUnits(baseUnits: string | number): string;
+  baseUnitsToBigUnits(baseUnits: string | number, chex?: boolean): string;
```

Comment:
please revert, we should not be adding something coin specific. there are other ways to determine the asset if it comes down to it (like checking `this.name` in the coin class)